src/classes/UTIL_OrgTelemetry_SObject_BATCH_TEST.cls

```diff
@@ -70,6 +70,9 @@ private class UTIL_OrgTelemetry_SObject_BATCH_TEST {
         opps.addAll(buildOpps(12, fieldValues));
 
         insert opps;
+
+        // Build data import records
```

Comment:
This comment probably isn't necessary, it's clear from the method name what's happening here